lesson_5/index5.html

```diff
@@ -69,5 +74,47 @@ <h1>Lesson_4</h1>
                 <td>0</td>
             </tr>
         </table>
+        <br>
+        <br>
+        <br>
+        <caption>Table 2</caption>
```

Comment:
**caption** tag must be inside of **table** tag

Reply:
ok